android/java/res/layout/wallet_balance_layout.xml

```diff
@@ -64,7 +63,7 @@
         android:scaleType="fitXY"
         app:srcCompat="@drawable/tooltip_arrow"
         android:contentDescription="@null"
-        android:layout_marginEnd="16dp"
+        android:layout_marginEnd="46dp"
```

Comment:
@sujitacharya2005 can you check on smaller screen, if the margin works there ?

Reply:
This is working fine in my small screen device.
This is for triange position above estimated tooltip view.